core/src/main/java/org/mskcc/cbio/oncokb/model/IndicatorQueryResp.java

```diff
@@ -1,42 +1,76 @@
 package org.mskcc.cbio.oncokb.model;
 
-
 import org.mskcc.cbio.oncokb.apiModels.Implication;
 import org.mskcc.cbio.oncokb.apiModels.MutationEffectResp;
 
+import io.swagger.annotations.ApiModelProperty;
+
 import java.util.ArrayList;
 import java.util.List;
 
-
-/**
- * TumorType generated by hbm2java
- */
-
 public class IndicatorQueryResp implements java.io.Serializable {
     private Query query;
-    private Boolean geneExist;
-    private Boolean variantExist;
-    private Boolean alleleExist;
+
+    @ApiModelProperty(value = "Indicates whether the gene is an OncoKB cancer gene")
+    private Boolean geneExist = false;
+
+    @ApiModelProperty(value = "Indicates whether an exact match for the queried variant is curated")
+    private Boolean variantExist = false;
+
+    @ApiModelProperty(value = "Indicates whether the alternate allele has been curated. See SOP Protocol 9.1")
+    private Boolean alleleExist = false;
+
+    @ApiModelProperty(value = "The oncogenicity status of the variant. Defaulted to \"Unknown\".", allowableValues = "Oncogenic, Likely Oncogenic, Likely Neutral, Inconclusive, Resistance, Unknown")
     private String oncogenic;
+
     private MutationEffectResp mutationEffect;
+
+    @ApiModelProperty(value = "(Nullable) The highest sensitivity level from a list of therapeutic evidences.", allowableValues = "LEVEL_1, LEVEL_2, LEVEL_3A, LEVEL_3B, LEVEL_4")
     private LevelOfEvidence highestSensitiveLevel;
+
+    @ApiModelProperty(value = "(Nullable) The highest resistance level from a list of therapeutic evidences.", allowableValues = "LEVEL_R1, LEVEL_R2")
     private LevelOfEvidence highestResistanceLevel;
+
+    @ApiModelProperty(value = "(Nullable) The highest diagnostic level from a list of diagnostic evidences.", allowableValues = "LEVEL_Dx1, LEVEL_Dx2, LEVEL_Dx3.")
     private LevelOfEvidence highestDiagnosticImplicationLevel;
+
+    @ApiModelProperty(value = "(Nullable) The highest prognostic level from a list of prognostic evidences.", allowableValues = "LEVEL_Px1, LEVEL_Px2, LEVEL_Px3")
     private LevelOfEvidence highestPrognosticImplicationLevel;
+
+    @ApiModelProperty(value = "(Nullable) The highest FDA level from a list of therapeutic evidences.", allowableValues = "LEVEL_Fda1, LEVEL_Fda2, LEVEL_Fda3")
     private LevelOfEvidence highestFdaLevel;
+
+    @ApiModelProperty(value = "DEPRECATED", allowableValues = "")
     private List<LevelOfEvidence> otherSignificantSensitiveLevels = new ArrayList<>();
+    @ApiModelProperty(value = "DEPRECATED", allowableValues = "")
     private List<LevelOfEvidence> otherSignificantResistanceLevels = new ArrayList<>();
-    private Boolean VUS;
-    private Boolean hotspot;
+
+    @ApiModelProperty(value = "Indicated whether it is a variant of unknown significance (investigated and data not found). See SOP Sub-Protocol 3.2")
```

Comment:
link to SOP if possible

Reply:
Links to specific sections at sop.oncokb.org is not possible.